src/System.Runtime.Serialization.Json/tests/ReflectionOnly/System.Runtime.Serialization.Json.ReflectionOnly.Tests.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);ReflectionOnly</DefineConstants>
+    <RootNamespace>System.Runtime.Serialization.Json.ReflectionOnly.Tests</RootNamespace>
+    <AssemblyName>System.Runtime.Serialization.ReflectionOnly.Json.Tests</AssemblyName>
```

Comment:
Do you like to keep the AssemblyName consistent with RootNamespace or is this intentional?